### ISSUE TITLE
Format all code before major release

### ISSUE THREAD

**Issue by Enet4** [bioinformatics-ua/dicoogle#319]
I propose that we define a coding style for Java code and stick to that.

For JavaScript code, we could use [prettier](https://github.com/prettier/prettier).

For Java code, I propose we look for a Maven plugin.

Update: Formatting with Prettier is already configured in #354. Now we need something for the Java code. Perhaps a Maven plugin?

**Comment by rlebre** [comment]
I found a maven plugin suitable for what we want. Perhaps [google-java-format](https://github.com/google/google-java-format). It can be integrated through [googleformatter-maven-plugin](https://github.com/talios/googleformatter-maven-plugin).

Example (as in README file):
```xml
    <plugin>
      <groupId>com.theoryinpractise</groupId>
      <artifactId>googleformatter-maven-plugin</artifactId>
      <version>1.7.3</version>
      <executions>
        <execution>
          <id>reformat-sources</id>
          <configuration>
            <includeStale>false</includeStale>
            <style>GOOGLE</style>
            <formatMain>true</formatMain>
            <formatTest>true</formatTest>
            <filterModified>false</filterModified>
            <skip>false</skip>
            <fixImports>false</fixImports>
            <maxLineLength>100</maxLineLength>
          </configuration>
          <goals>
            <goal>format</goal>
          </goals>
          <phase>process-sources</phase>
        </execution>
      </executions>
    </plugin>
```

**Comment by Enet4** [comment]
Looks good, as long as the license header remains untouched and we all agree to the imposed style. By the looks of it, the Google Java Style will touch nearly all of the Java source files, since it includes 2-space indentation.

Are there other code formatters we could use?

**Comment by rlebre** [comment]
I found also [formatter-maven-plugin](https://code.revelc.net/formatter-maven-plugin)

**Comment by bastiao** [comment]
Fixed in bdf92d6f526532b7655896e725a58428b8eb76fc
